packages/graphql/src/translate/cypher-builder/variables/Literal.ts

```diff
@@ -18,22 +18,35 @@
  */
 
 import type { CypherEnvironment } from "../Environment";
+import type { CypherCompilable } from "../types";
 
-type LiteralValues = string | number | boolean | null;
+type LiteralValue = string | number | boolean | null | Array<LiteralValue>;
 
 /** Represents a literal value, it is not a variable */
-export class Literal<T = LiteralValues | LiteralValues[]> {
+export class Literal<T extends LiteralValue = any> implements CypherCompilable {
     public value: T;
 
     constructor(value: T) {
         this.value = value;
     }
 
     public getCypher(_env: CypherEnvironment): string {
```

Comment:
question (non-block): What is the `_env` for?

Reply:
It is part of the expected interface for an element that can be compiled into cypher. Is the environment that holds reference to the variables.

In this case it is not needed, but because all of these instances will be processed the same way, I added to keep consistency